lib/widgets/forest_park_map.dart

```diff
@@ -140,7 +153,7 @@ class _ForestParkMapState extends ConsumerState<ForestParkMap> with WidgetsBindi
                     .updateZoom(position.zoom!));
           }
           if (hasGesture) {
-            ref.read(followOnLocationProvider.notifier).update((state) => FollowOnLocationUpdate.never);
+            ref.read(followOnLocationTargetProvider.notifier).state = FollowOnLocationTargetState.currentLocation;
```

Comment:
This should set it to .none, not .currentLocation (currently when you try to move the map while you're on the current location, it will jump back to the currentLocation)